canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -39,8 +39,9 @@
 		margin-bottom: $spacing-02;
 		margin-right: $spacing-04;
 		display: grid;
-		grid-template-columns: 1fr 8fr;
+		grid-template-columns: 1fr 11fr;
 		grid-template-rows: auto;
+		align-items: center;
```

Comment:
This does center it but you'll also need to add a bottom margin to the tooltip to match the bottom margin of the radio button.  I think it's $spacing-01

Reply:
Updated it.